src/systemctl/systemctl.c

```diff
@@ -4660,35 +4676,30 @@ static int show_one(
 
         r = 0;
 
-        if (!show_properties) {
-                if (streq(verb, "help"))
-                        show_unit_help(&info);
+        if (show_properties) {
+                char **pp;
+
+                STRV_FOREACH(pp, arg_properties) {
+                        if (!set_contains(found_properties, *pp)) {
+                                log_warning("Property %s does not exist.", *pp);
+                                r = -ENXIO;
+                        }
```

Comment:
we use the lsb exit codes only for "systemctl status", but this code is only used for "systemctl show", hence I just returned normal error codes in this case.